src/Microsoft/Silk.NET.DirectComposition/Structs/TagCOMPOSITIONFRAMESTATS.gen.cs

```diff
@@ -17,9 +17,9 @@
 namespace Silk.NET.DirectComposition
 {
     [NativeName("Name", "tagCOMPOSITION_FRAME_STATS")]
-    public unsafe partial struct CompositionFrameStats
+    public unsafe partial struct TagCOMPOSITIONFRAMESTATS
```

Comment:
How come this has been reverted?

Reply:
that name was manual, so since i regenerated it borked again, i'll have to fix it manually

Reply:
Didn't we have a rename entry for this in the json? If not, sucks that we (I) didn't notice that...

Reply:
Added a manual rename in the json, i think whoever renamed it either forgot that part of the generator existed or forgot to commit